src/main/java/com/derongan/minecraft/deeperworld/listeners/MovementListener.kt

```diff
@@ -58,6 +58,7 @@ object MovementListener : Listener {
                 && DeeperConfig.data.damageOutsideSections > 0.0
                 && (player.gameMode == GameMode.SURVIVAL || player.gameMode == GameMode.ADVENTURE)
             ) {
+                if(player.location.world !in DeeperConfig.data.worlds) return
```

Comment:
Add this to the if statement above. Since we're at it, I'd appreciate it if you could reoder the statements so the damageOutsideSections check goes first (it's the fastest), followed by the gamemode check, followed by this, followed by the damageExcluded worlds check.

Also rewrite that last check to also use `!in` instead of `.contains()`